page.codeberg.DrRac27.Karoto.yml

```diff
@@ -0,0 +1,41 @@
+app-id: page.codeberg.DrRac27.Karoto
+runtime: org.kde.Platform
+runtime-version: "6.4"
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: "6.4"
+command: karoto
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--device=dri"
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - '*.la'
+  - '*.a'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: karoto
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-deps --prefix=${FLATPAK_DEST} karoto-0.1-py3-none-any.whl
+      - install -Dm644 packaging/karoto.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 packaging/karoto.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
```

Comment:
if you use `rename-desktop-file` instead of renaming on install then it will also fix the appstream file "launchable" which is why it doesn't find the icon.

Reply:
Thanks, I was already writing another `sed` line for the "launchable" as I saw your notification popping up ^^